lib/Template.js

```diff
@@ -202,4 +202,43 @@ module.exports = class Template {
 		}
 		return source;
 	}
+
+	static renderHTMLChunk(chunk, filterFn, moduleTemplate, dependencyTemplates) {
```

Comment:
Shouldn't this be `renderHtmlChunk`? (etc.)

It's `stats.toJson()`, `JsonGenerator`, `JsonParser`, etc., so acronyms are written in sentence case.